File: courses/csharp-introduction/feet_to_inches_converter.cs
using System;

public class Program
{
    static void Main(string[] args)
    {
        double foot = Convert.ToDouble(Console.ReadLine());
        Converter(foot);
    }

    // your code goes here
    static void Converter(double foot)
    {
        double inch = foot * 12;
        Console.WriteLine(inch);
    }
}

File: courses/java-introduction/feet_to_inches_converter.java
import java.util.Scanner;

public class Program {
    // your code goes here
    static void convert(double num) {
        double inches = num * 12;
        System.out.println(inches);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double num = sc.nextDouble();
        convert(num);
    }
}